eex/datalayer.py

```diff
@@ -1078,6 +1078,41 @@ def list_term_parameters(self, order):
 
         return self._terms[order]
 
+    def remove_term_parameter(self, order, uid):
+        """
+        Removes parameter from datalayer based on term order and uid. This paramater must have a term count of 0.
+
+        Parameters
+        -------------
+        order : int
+            The order of the functional form (2, 3, 4, ...)
+        uid: int
+            The uid of the term to be removed.
+
+
+        Return
+        --------------
+            Returns True if successful
+        """
+
+        # Check that order exists in datalayer
+        if order not in list(self._terms.keys()):
+            raise KeyError("No terms with order %s exist" % order)
+
+        # Check that uid is int and exists in dl.
+        if uid not in list(self._terms[order].keys()):
```

Comment:
`list` should not be needed here. `dict.keys()` returns effectively a set. Also, `if uid not in self._terms[order]` will work equally as well and should be a bit simpler!